src/components/Stepper/StepConnector.test.tsx

```diff
@@ -0,0 +1,159 @@
+
+  const { getByTestId } = render(
+    <StepperContextProvider value={mockContextValue}>
+      <StepConnector status='active' direction='horizontal' labelHeight={10} />
+    </StepperContextProvider>
+  )
+
+  const connector = getByTestId('step-connector')
+
+  expect(getComputedStyle(connector).width).toBe('calc(100% - 1rem)')
+  expect(getComputedStyle(connector).height).toBe('')
+  expect(getComputedStyle(connector).borderTopWidth).toBe('2px')
+  expect(getComputedStyle(connector).borderRightWidth).toBe('')
+  expect(getComputedStyle(connector).top).toBe('calc(0.75rem - 1px)')
+  expect(getComputedStyle(connector).left).toBe('calc(50% + 0.5rem)')
+})
+
+it('should apply correct styles when direction is vertical', () => {
+  const mockContextValue = createMockStepperContext()
+
+  const { getByTestId } = render(
+    <StepperContextProvider value={mockContextValue}>
+      <StepConnector status='active' direction='vertical' labelHeight={10} />
+    </StepperContextProvider>
+  )
+
+  const connector = getByTestId('step-connector')
+
+  expect(getComputedStyle(connector).height).toBe('calc(100% - 1rem)')
+  expect(getComputedStyle(connector).width).toBe('')
+  expect(getComputedStyle(connector).borderRightWidth).toBe('2px')
+  expect(getComputedStyle(connector).borderTopWidth).toBe('')
+  expect(getComputedStyle(connector).top).toBe('calc(10px / 2 + 0.5rem)')
+  expect(getComputedStyle(connector).left).toBe('calc(0.75rem - 1px)')
+})
+
+it('should use gap from context when defined', () => {
+  const gap = 2
+  const mockContextValue = createMockStepperContext({ gap })
+
+  const { getByTestId } = render(
+    <StepperContextProvider value={mockContextValue}>
+      <StepConnector status='active' direction='horizontal' labelHeight={10} />
+    </StepperContextProvider>
+  )
+
+  const connector = getByTestId('step-connector')
+
+  expect(getComputedStyle(connector).width).toBe(`calc(100% + ${gap}rem - 1rem)`)
+})
+
+it('should change border when status is active', () => {
+  const mockContextValue = createMockStepperContext()
+
+  const { getByTestId } = render(
+    <StepperContextProvider value={mockContextValue}>
+      <StepConnector status='active' direction='horizontal' labelHeight={10} />
+    </StepperContextProvider>
+  )
+
+  const connector = getByTestId('step-connector')
+
+  expect(getComputedStyle(connector).borderTopStyle).toBe('solid')
+  expect(getComputedStyle(connector).borderTopColor).toBe('#0069D0')
+})
+
+it('should change border when status is completed', () => {
+  const mockContextValue = createMockStepperContext()
+
+  const { getByTestId } = render(
+    <StepperContextProvider value={mockContextValue}>
+      <StepConnector status='completed' direction='horizontal' labelHeight={10} />
+    </StepperContextProvider>
+  )
+
+  const connector = getByTestId('step-connector')
+
+  expect(getComputedStyle(connector).borderTopStyle).toBe('solid')
+  expect(getComputedStyle(connector).borderTopColor).toBe('#0069D0')
+})
```

Comment:
Is it correct that there’s no difference in the StepConnector component between the completed and active status?